torchaudio/transforms/_transforms.py

```diff
@@ -1243,6 +1243,72 @@ def __init__(self, time_mask_param: int, iid_masks: bool = False, p: float = 1.0
         super(TimeMasking, self).__init__(time_mask_param, 2, iid_masks, p=p)
 
 
+class SpecAugment(torch.nn.Module):
+    r"""Apply time and frequency masking to a spectrogram.
+
+    Args:
+        n_time_masks (int): Number of time masks.
+        time_mask_param (int): Maximum possible length of the time mask.
+        n_freq_masks (int): Number of frequency masks.
+        freq_mask_param (int): Maximum possible length of the frequency mask.
+        iid_masks (bool, optional): Applies iid masks to each of the examples in the batch dimension.
+            This option is applicable only when the input tensor is 4D. (Default: ``True``)
+        p (float, optional): maximum proportion of time steps that can be masked.
+            Must be within range [0.0, 1.0]. (Default: 1.0)
```

Comment:
I feel those parameters can be put in `forward` method so that users have more flexibility to use the same module. Otherwise they have to create new `SpecAug` modules with different parameters, WDYT?

Reply:
oh I think it's ok. given a specific certain training task, these parameters shouldn't change. so one instance of the class is enough